mantis/cli/apply_affine.py

```diff
@@ -108,7 +108,8 @@ def apply_affine(
         )
         # Overwrite the previous target shape
         Z_target, Y_target, X_target = cropped_target_shape_zyx[-3:]
-        click.echo(f'Shape of cropped output dataset: {target_shape_zyx}\n')
+        cropped_target_shape_zyx = Z_target, Y_target, X_target
```

Comment:
This doesn't make sense to me. Is `cropped_target_shape_zyx` more than 3D such that you only need to take the last 3 dims? These two lines look redundant.

Reply:
Z_target,Y_target,X_target were just making sure the target variables were replaced. They need to be updated for the `create_empty_hcs_zarr` but these can be changed. 

I'll refactor the code so that we use less variables across the code.